resources/definitions/flashforge_base.def.json

```
@@ -0,0 +1,44 @@
+{
+    "version": 2,
+    "name": "Dreamer NX",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": false,
+        "author": "Egon",
+        "manufacturer": "Flashforge",
+        "file_formats": "text/x-gcode",
+        "first_start_actions": ["MachineSettingsAction"],
+        "machine_extruder_trains":
+        {
+            "0": "ff_dreamernx_extruder_0"
```

Minor thing, but please rename the extruder to include the full name of the printer
```suggestion
            "0": "flashforge_dreamernx_extruder_0"
```

Also rename the file `resources/extruders/ff_dreamernx_extruder_0.def.json` to `resources/extruders/flashforge_dreamernx_extruder_0.def.json`